bin/travis-build.bash

```diff
@@ -78,9 +78,23 @@ paster db init -c test-core.ini
 cd ..
 echo "-----------------------------------------------------------------"
 echo "Installing Harvester"
-git clone https://github.com/ckan/ckanext-harvest
+
+git clone https://github.com/GSA/ckanext-harvest
```

Comment:
Shouldn't this be ckan/ckanext-harvest for 2.8 and GSA/ckanext-harvest for 2.3?

Reply:
@adborden We are waiting for [a PR](https://github.com/GSA/ckanext-harvest/pull/10) to be merged at upstream.
This is related to our extra notifications.
Until that happens we have our fork using the branch `datagov-catalog`.